builtin/providers/azurerm/resource_arm_loadbalancer.go

```diff
@@ -0,0 +1,295 @@
+package azurerm
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/arm/network"
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/jen20/riviera/azure"
+)
+
+func resourceArmLoadbalancer() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLoadbalancerCreate,
+		Read:   resourceArmLoadbalancerRead,
+		Update: resourceArmLoadbalancerCreate,
+		Delete: resourceArmLoadbalancerDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": {
+				Type:      schema.TypeString,
+				Required:  true,
+				ForceNew:  true,
+				StateFunc: azureRMNormalizeLocation,
+			},
+
+			"resource_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"frontend_ip_configuration": {
+				Type:     schema.TypeSet,
```

Comment:
Do we strictly need this to be a Set? Is there any reason not to have it as a list?